docs/source/redhat-opens-install.rst

```diff
@@ -312,17 +312,10 @@ On each control node, perform the following steps:
    OpenStack packages and to ``dnsmasq``.  (OpenStack updates are not needed
    for Liberty.)
 
-3. Edit the ``/etc/neutron/plugins/ml2/ml2_conf.ini`` file.  In the `[ml2]`
-   section:
+3. Edit the ``/etc/neutron/neutron.conf`` file.  In the `[DEFAULT]` section:
 
-   -  Find the ``type_drivers`` setting and change it to read
-      ``type_drivers = local, flat``.
-   -  Find the ``tenant_network_types`` setting and change it to read
-      ``tenant_network_types = local``.
-   -  Find the ``mechanism_drivers`` setting and change it to read
-      ``mechanism_drivers = calico``.
-
-4. Edit the ``/etc/neutron/neutron.conf`` file.  In the `[DEFAULT]` section:
+   -  Find the line beginning with ``core_plugin``, and change it to read
+      ``core_plugin = calico.openstack.core_calico.CalicoCorePlugin``.
```

Comment:
Sure, it can wait.  My personal opinion is that it's nice to have less code paths to test, and also supporting both will add (slightly) more code.  To handle both cases, the code that queries for floating IPs would have to be disabled somehow when the core plugin isn't loaded.  I'd rather not do a try/except around it, so that failures aren't ignored when they should actually be working (when the calico core plugin is actually loaded).